src/group/endpoints/group.search.ts

```diff
@@ -24,6 +24,12 @@ export default class SearchGroup extends Endpoint {
     });
   }
 
+  private static escapeRegex(str: string): string {
+    if (typeof str !== 'string') return str; // TODO: Decide whether this throws an error or not
```

Comment:
The `partial` must be a string. It can also be a string that contains only numbers.